docs/get-started/how-it-works.mdx

```diff
@@ -30,46 +38,93 @@ A commons' data is stored on its users' devices using **pockets**.
 
 The address is publicly shareable information, and the secret should be kept, well, secret. A keypair's address or secret can never be changed, so keep it safe!
 
-When a public name is put into a document's path, prefixed with a tilde (e.g. `/about/@suzy.bzfkbawuj2cdxsxilvso4a2xczmcx6char4pqd2tjhdmcoy5i4ebq/displayName.txt`), then _only_ the user with that identity can write to that document.
+When a public name is put into a document's path, prefixed with a tilde (e.g. `/about/~@suzy.bzfkbawuj2cdxsxilvso4a2xczmcx6char4pqd2tjhdmcoy5i4ebq/displayName.txt`), then _only_ the user with that identity can write to that document.  Otherwise anyone can edit the document.
 
 But what about the ways we express our identity, like names, pronouns, or avatars?
 
-This kind of information is stored as data within the commons you use. This has two big benefits: your identity can freely fluctuate between the spaces you use; and you can express your identity with as little or as much detail as you want!
+This kind of information is stored as documents within the commons, e.g. in your ["about me"](https://github.com/earthstar-project/earthstar/wiki/Standard-paths-and-data-formats-used-by-apps#about-author-profile-info) document. This has two big benefits: your identity can freely fluctuate between the spaces you use; and you can express your identity with as little or as much detail as you want!
 
 ## Applications
 
-Applications are just programs which write and read through a pocket's contents and re-present it as a friendly user interface. For example, imagine a program which reads a folder full of images and displays them as a gallery.
+Applications are just programs which write and read through a pocket's documents and re-present it as a friendly user interface. For example, imagine a program which reads a folder full of images and displays them as a gallery.
 
 Earthstar has **no opinion** on how applications are created, and provides a Javascript API that you can use in your vanilla JS, jQuery, React, Svelte, Vue, Angular, or whatever app.
 
+Earthstar apps can typically run in browsers, or can be native apps.  They just need to be able to talk to the other peers to sync data.
+
+## Servers & Peers
+
+> Pocket Servers: previously known as: *pub servers*, *cloud pockets*
+
+Most laptops and phones are not able to connect directly to each other, so we need some third space they can both connect to where data can be temporarily stashed for syncing.
+
+These are called **Pocket Servers**.  You can run your own pocket server(s) for your community on a small device like a raspberry pi, or hosted in the cloud.
+
+To run your own pocket server, see instructions in the [earthstar-pub](https://github.com/earthstar-project/earthstar-pub/#how-to-deploy) repo.
```

Comment:
Let's remove this paragraph: it refers to the v1 ecosystem where the rest of the site is focused on v2. I will add something to this site when we have a stopgap pub (which is part of the grant milestone this site belongs to).

Reply:
Do you want to remove the whole section about peers and pubs?  It seems important to help people understand how things work, or will work.

Maybe if this was written in future tense, "you will be able to run..."?

Reply:
No, just the line 'to run your own pocket server, see instructions...'. Everything else you added is important!

Reply:
Ok, I removed the sentence that mentioned `earthstar-pub`.